app/src/main/res/layout/fragment_main_page.xml

```diff
@@ -16,67 +17,269 @@
+            app:layout_behavior="@string/appbar_scrolling_view_behavior"
             >
-            <TextView
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:text="@string/title_featured_product"
-                android:textColor="@color/colorAccent"
-                android:textSize="@dimen/sp_15"
-                />
-            <TextView
-                android:layout_width="wrap_content"
+            <RelativeLayout
+                android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:layout_alignParentRight="true"
-                android:onClick="@{() -> viewModel.onSeeMoreProductClick()}"
-                android:text="@string/action_see_more"
-                android:textColor="@android:color/holo_blue_dark"
-                android:textSize="@dimen/sp_15"
-                />
-        </RelativeLayout>
-        <android.support.v7.widget.RecyclerView
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            app:layoutManager="@{LayoutManagers.grid(mainPage.getResources.getInteger(R.integer.item_col))}"
-            app:recyclerAdapter="@{viewModel.productAdapter}"
-            />
-    </LinearLayout>
+                >
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:background="@color/light_gray"
+                    android:orientation="vertical"
+                    >
+                    <TextView
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:background="@drawable/border_layout"
+                        android:drawableLeft="@drawable/ic_label"
+                        android:drawablePadding="@dimen/dp_5"
+                        android:drawableStart="@drawable/ic_label"
+                        android:fontFamily="@string/font_serif"
+                        android:gravity="center_vertical"
+                        android:padding="@dimen/dp_5"
+                        android:text="@string/category"
+                        android:textColor="@color/color_orange_mos"
+                        android:textSize="@dimen/sp_17"
+                        android:textStyle="bold"
+                        tools:ignore="UnusedAttribute"
+                        />
+                    <LinearLayout
+                        android:layout_width="match_parent"
+                        android:layout_height="match_parent"
+                        android:layout_marginBottom="@dimen/dp_5"
+                        android:layout_marginTop="@dimen/dp_5"
+                        android:orientation="vertical"
+                        >
+                        <LinearLayout
```

Comment:
WARNING: [UselessParent] This `LinearLayout` layout or its `LinearLayout` parent is useless
 A layout with children that has no siblings, is not a scrollview or a root layout, and does not have a background, can be removed and have its children moved directly into the parent for a flatter and more efficient layout hierarchy.